2/contrib/openshift/jenkins-version.txt

```diff
@@ -1 +1 @@
-2.440.3
+2.452.4
```

Comment:
LST version 2.462.1 also has the fix.
https://www.jenkins.io/security/advisory/2024-08-07/

Reply:
Ok, changed to 2.462.1